backend/core/Extensions/GeometryExtensions.cs

```diff
@@ -0,0 +1,80 @@
+using NetTopologySuite.Geometries;
+
+namespace Pims.Core.Extensions
+{
+    /// <summary>
+    /// GeometryExtensions static class, provides extension methods for geometry objects.
+    /// </summary>
+    public static class GeometryExtensions
+    {
+        /// <summary>
+        /// Convert an Envelope to a Polygon.
+        /// </summary>
+        /// <param name="envelope"></param>
+        /// <returns></returns>
+        public static Polygon ToPolygon(this Envelope envelope)
+        {
+            if (envelope != null && envelope.MinX <= envelope.MaxX && envelope.MinY <= envelope.MaxY)
+            {
+                var pfactory = new GeometryFactory();
+                var bounds = new []
+                {
+                    new Coordinate(envelope.MinX, envelope.MinY),
+                    new Coordinate(envelope.MaxX, envelope.MinY),
+                    new Coordinate(envelope.MaxX, envelope.MaxY),
+                    new Coordinate(envelope.MinX, envelope.MaxY),
+                    new Coordinate(envelope.MinX, envelope.MinY),
+                };
+
+                var poly = pfactory.CreatePolygon(new LinearRing(bounds));
+                poly.SRID = 4326;
+                return poly;
+            }
+            return null;
+        }
+
+        /// <summary>
+        /// Convert an array of Coordinate to a Polygon.
+        /// If the coordinate array is only a length of two it will be treated like a boundary.
+        /// If the coordinate array does not self close, it will by default close with the inital coordinates.
+        /// The coordinate array should be expressed counter-clockwise.
+        /// </summary>
+        /// <param name="coords"></param>
+        /// <returns></returns>
+        public static Polygon ToPolygon(this Coordinate[] coords)
```

Comment:
Centralize this function so that all the queries that use it can be updated in one place.